Terminal.Gui/Application/Application.Run.cs

```diff
@@ -26,13 +26,13 @@ public static Key QuitKey
     private static Key _arrangeKey = Key.F5.WithCtrl; // Resources/config.json overrides
 
     /// <summary>Gets or sets the key to activate arranging views using the keyboard.</summary>
-    [SerializableConfigurationProperty (Scope = typeof (SettingsScope))]
+    [ConfigurationProperty (Scope = typeof (SettingsScope))]
     public static Key ArrangeKey
     {
         get => _arrangeKey;
         set
         {
-            if (_arrangeKey != value)
+            //if (_arrangeKey != value)
```

Comment:
Is this meant to be commented out? The curly bracers remain...

Reply:
Yes. This will be fixed as part of fixing #2871 where `INotifyPropertyChanged` will come into play.